src/documents/CoreAPI.md

```diff
@@ -0,0 +1,635 @@
+currently empty paths leading to the global namespace root. Calls to `fluid.registerNamespace` are very common at the start of Infusion-aware `.js` files - they are analogous to `require` statements,
+only they do not cause any code to be loaded by themselves - but they can grant access to implementations which have been loaded by others, or else provide an intermediate node at which to export
+implementations to others.
+
+* `path {String|Array of String}` The global path at which the namespace is to be allocated
+* Returns: `{Any}` Any current value held at the supplied path - or a freshly allocated `{}` to be held at that path if it was previously empty
+
+### fluid.getGlobalValue(path)
+
+Returns any value held at a particular global path. This may be an object or a function, depending on what has been stored there. 
+
+* `path {String|Array of String}` The global path from which the value is to be fetched
+* Returns: `{Any}` The value that was stored at the path, or `undefined` if there is none. 
+
+### fluid.setGlobalValue(path, value)
+
+Write a value to a particular global path. Users are strongly discouraged from storing any mutable information at these paths, but it is accepted that in certain integration scenarios there can be little alternative. 
+Please choose a properly qualified global path name that identifies your project (perhaps, derived from a [domain name](https://en.wikipedia.org/wiki/Domain_name) or [npm package name](https://docs.npmjs.com/misc/faq#why-no-namespaces)
+that you have control over. The [Java package naming conventions](https://en.wikipedia.org/wiki/Java_package#Package_naming_conventions) have proved effective at heading off such problems in the past and 
+we intend that Infusion global names should be handled in an analogous way. 
+
+* `path {String|Array of String}` The global path at which the value is to be set. 
+* `value {Any}` The value to be written to the global path. This should in general be a function or other immutable value, and an existing value should not be overwritten. This rule can be overlooked in a genuine architectural emergency.
+
+### fluid.invokeGlobalFunction(functionPath, args)
+
+Invokes a function held at a particular global path with the supplied arguments. Equivalent to `fluid.getGlobalValue` followed by `Function.apply(null, args)`.
+
+* `path {String|Array of String}` The global path holding the function which is to be invoked
+* `args {Array|Any}` The array of arguments to be supplied to the function. If `args` is not an array, `fluid.makeArray` will be called on it first. 
+* Returns: {Any} Any return value from the function which was invoked
+
+### fluid.invokeGradedFunction(name, spec)
+
+Invoke a global function by name and named arguments. A courtesy to allow declaratively encoded function calls
+to use named arguments, with names encoded in the defaults for a `fluid.function`, rather than bare arrays.
```

Comment:
Actually, the entire description of this function is a bit vague. I think it assumes a lot of knowledge on the part of the reader – knowledge the reader likely doesn't actually have. Perhaps back up a bit, and provide a bit more explanation?